doc/NITS.md

```diff
@@ -71,27 +80,32 @@ Note: `XX` means the name of the ITS you are building.
    D$(N)SALV
    ```
 
-5. Merge the ITS binary with DDT and (N)SALV.
+## Step 5. Merge the ITS binary with DDT and (N)SALV
 
    There are two options for doing this.  The normal way is to reboot and do it in DSKDMP.  The other way is to do it in timesharing DDT.
 
-   - DSKDMP method.  Use this unless you have a good reason not to.
+### DSKDMP method
+Use this unless you have a good reason not to.
+
+  1. Shut down ITS with LOCK.  Reboot into DSKDMP.  Use the new DSKDMP if you made one above.
+  2. Load DDT: <code>L<kbd>$</kbd>DDT</code>
+  3. Give ITS and its symbols to DDT: <code>T<kbd>$</kbd>NITS BIN</code>
+  4. You're now in DDT.  Exit back to DSKDMP: <code><kbd>$</kbd>U</code>
+  5. Merge in (N)SALV.  For KA10: <code>M<kbd>$</kbd>SALV BIN</code>  For KS10: <code>M<kbd>$</kbd>NSALV BIN</code>
```

Comment:
Should include KL10 too, right?

Reply:
Maybe the text needs a larger overhaul for to add KL10 instructions.

Reply:
Ah, sorry, I was improving the formatting on the markdown, but had to change indentation on this block to prevent it from being parsed as a blockquote. This one has no changes in the text.